src/observability/publisher.go

```diff
@@ -58,32 +58,40 @@ func ProcessSystemSummary() {
 
 func getAggregatedSummaryMap(tempSummarizerMap map[types.SystemSummary]types.SysSummaryTimeCount) map[types.SystemSummary]types.SysSummaryTimeCount {
```

Comment:
```suggestion
func aggregateFileSummary(summary map[types.SystemSummary]types.SysSummaryTimeCount)  {
```
Since this function modify the input, there no need for a return type.